### ISSUE TITLE
Slather does not have an --input-format argument

### ISSUE THREAD

**Issue by taiheng** [KrauseFx/fastlane#650]
As far as I can tell slather does not have this argument which is required for the slather step.

Environment:
slather 1.8.1 installed via ruby gem
fastlane 1.30.2

```
Usage:
    slather coverage [OPTIONS] [xcodeproj]

Parameters:
    [xcodeproj]                   Path to the xcodeproj

Options:
    --travis, -t                  Indicate that the builds are running on Travis CI
    --circleci                    Indicate that the builds are running on CircleCI
    --jenkins                     Indicate that the builds are running on Jenkins
    --coveralls, -c               Post coverage results to coveralls
    --simple-output, -s           Output coverage results to the terminal
    --gutter-json, -g             Output coverage results as Gutter JSON format
    --cobertura-xml, -x           Output coverage results as Cobertura XML format
    --html, -h                    Output coverage results as static html pages
    --show                        Indicate that the static html pages will open automatically
    --build-directory, -b BUILD_DIRECTORY The directory where gcno files will be written to. Defaults to derived data.
    --source-directory SOURCE_DIRECTORY The directory where your source files are located.
    --output-directory OUTPUT_DIRECTORY The directory where your Cobertura XML report will be written to.
    --ignore, -i IGNORE           ignore files conforming to a path
    -h, --help                    print help
```

Running this step without `input_format` causes fastlane to ask for --input-format interactively and will still fail even if no input is provided:
```
[10:54:48]: ---------------------
[10:54:48]: --- Step: slather ---
[10:54:48]: ---------------------
[10:54:48]: To not be asked about this value, you can specify it using 'input_format'
The input format that slather should look for:
[10:54:54]: [SHELL COMMAND]: slather coverage  --build-directory ./ --input-format  --scheme XXX XXX.xcodeproj
[10:54:54]: [SHELL]: ERROR: Unrecognised option '--input-format'
[10:54:54]: [SHELL]:
[10:54:54]: [SHELL]: See: 'slather coverage --help'
```

**Comment by pprochazka72** [comment]
@taiheng I experience the same issue

**Comment by xfreebird** [comment]
@taiheng that is because officially that option is not yet merged [#99](https://github.com/venmo/slather/pull/99) in the master branch of slather

**Comment by taiheng** [comment]
I do not know, but perhaps --input-format and --build-directory should be left as optional in that action to match what slather requires. That or this should be documented a little better in the Slather action page.
